### PR TITLE
resolve lasso norm deprecation problem in newer sklearn versions

### LINKED ISSUE #28: Deprecated Python Feature used in chimes_lsq
See the attached image for the error in question, obtained when running the lsq2 portion of the ChIMES_LSQ lsq test suite. @melodyyzh, can you take a look at this and see if you can update our code accordingly?

Image

### PR DESCRIPTION
**Pull request by melodyyzh**
## Description
Scikit-learn Lasso function deprecates the `normalize` command in version 1.4. Therefore, we need to change our CHIMES_LSQ implementation accordingly. 
In the original implementation, sklearn sets default `normalize=True`; however, the `normalize` kwarg is ignored when the `fit_intercept=False`, which is what ChIMES_LSQ implemented. Therefore, ChIMES_LSQ does not normalize data. 
To accomondate the deprecation, I used the `make_pipeline` and `StandardScalar` functions in scikit-learn, with `with_mean=False, with_std=False` to achieve an equivalent purpose. 

## Motivation and context

Resolves #28 

## How has this been tested?

Use the `lsq2` test case folder in `test_suite-lsq`. I first ran `make generate` using the **pre-modified** `chimes_lsq.py` to generate the `correct_output` folder for the Stampede3 HPC. Then I copied over the folder and ran the `make all` command using the **modified** `chimes_lsq.py` to compare the param files generated using different solving algorithms. 

## Change log

```
reg = make_pipeline(preprocessing.StandardScaler(with_mean=False, with_std=False), 
                  linear_model.LassoLars(
                      alpha=args.alpha,
                      fit_intercept=False,
                      fit_path=False,
                      verbose=True,
                      max_iter=100000,
                      copy_X=False)
                  )
```

## Checklist:

- [x] I have ran the chimes-md and chimes-lsq test suites and attached the log files in this PR.   
- [x] I have reviewed the [**Contributor Guidelines**](https://github.com/LindseyLab-umich/chimes_lsq-LLfork/blob/main/doc/source/contributing.rst).
- [x] I agree with the terms of the [**ChIMES Contributor Agreement**](link).
- [x] My name is on the list of contributors (`path/to/file`) in the pull request source branch.

### CODE REVIEW

**Comment by melodyyzh** [comment]
Here I included the command line output while running the Makefile. This test is a bit different since I need to first run `make generate` to generate correct output for the current machine (Stampede3), then run `make all` to run the actual test and compare. 

I also included the `correct_output` for the LLNL machine and Stampede3, just for reference. If you run `vimdiff`, for the Lassolars file that we care about, the RMS force matches exactly. The coefficients have small differences, generally starting from at least the 4th decimal place onwards. The same applied for the rest of the algorithm files, with some exceptions (e.g., `params.dlasso.1.0e-04.txt`). 

RMS force error matching is a good indicator of the correct fit, and small differences in coefficients could be caused by the difference in HPC systems.
